app/workers/publish_classification_event_worker.rb

```diff
@@ -12,7 +12,7 @@ def perform(classification_id)
     if classification.complete?
       EventStream.push("classification",
         event_id: "classification-#{classification.id}",
-        event_time: "#{classification.updated_at}",
+        event_time: classification.updated_at.iso8601,
```

Comment:
Now you end up converting to a string, parsing it back, and then converting it to a string again. Can we not just require this to just be a Time instance, and always let EventStream call `.iso8601` on it?